contents/spotlight/startup-before-sunset-ai.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Startup Spotlight: How BeforeSunset AI mastered Product Hunt launches"
```

Comment:
```suggestion
title: "How BeforeSunset AI achieved 'Product of the Month' on Product Hunt"
```

Feel free to disregard, but if we have the custom image we can drop the "Startup Spotlight" kicker and go with something like this. Adding they were product of the month validates what "mastered" really means. That's a massive, eye-opening achievement, right?